larray/core/array.py

```diff
@@ -6841,6 +6837,18 @@ def create_sequential(axis, initial=0, inc=None, mult=1, func=None, axes=None, t
     >>> initial
     sex  M  F
          3  4
+    >>> create_sequential(year, initial, 1)
+    sex\\year  2016  2017  2018  2019
+           M     3     4     5     6
+           F     4     5     6     7
+    >>> create_sequential(year, initial, 1.0)
```

Comment:
this is one example too much IMO. It does not really help understanding what the function does. We probably want to have this example (using a float argument) as a unit test though.